libs/ui-components/src/components/common/LabelsView.tsx

```diff
@@ -1,14 +1,33 @@
 import * as React from 'react';
-
 import { Label, LabelGroup } from '@patternfly/react-core';
 
-const LabelsView = ({ prefix, labels }: { prefix: string; labels: Record<string, string | undefined> | undefined }) => {
+import { useTranslation } from '../../hooks/useTranslation';
+
+import './LabelsView.css';
+
+interface LabelsViewProps {
+  prefix: string;
+  labels: Record<string, string | undefined> | undefined;
+  variant?: 'default' | 'collapsed';
+}
+
+const LabelsView = ({ variant, prefix, labels }: LabelsViewProps) => {
+  const { t } = useTranslation();
   const labelItems = Object.entries(labels || {});
   if (labelItems.length === 0) {
     return '-';
   }
+
+  const isLimited = variant === 'collapsed';
   return (
-    <LabelGroup numLabels={5}>
+    <LabelGroup
+      numLabels={isLimited ? 2 : 5}
+      className={isLimited ? 'fctl-labels-view__collapsed' : undefined}
+      expandedText={t('Show less')}
+      // Label group will use this template and inject the correct count of labels.
+      // For it to work, the variable name must be "remaining"
+      collapsedText={t('{{remaining}} more', { remaining: 0 })}
```

Comment:
hmm, is this working now ? the `t` function will return a string `0 remaining` so I dont think PF will replace the 0.

I think it should be something like 
```suggestion
      collapsedText={'${remaining} '+t('more')}
```

`collapsedText` prop is is very i18n unfriendly :/

Reply:
Yes, it's a tricky one. It worked well when passing the pattern with `${remaining}`, but now I see it doesn't work with the proper `i18n` label.

I like your suggestion better to avoid confusing translators and make PF happy as well.